node/cmd/guardiand/node.go

```diff
@@ -904,16 +932,24 @@ func runNode(cmd *cobra.Command, args []string) {
 
 		logger.Info("Starting Terra watcher")
 		if err := supervisor.Run(ctx, "terrawatch",
-			terra.NewWatcher(*terraWS, *terraLCD, *terraContract, lockC, setC, chainObsvReqC[vaa.ChainIDTerra], common.ReadinessTerraSyncing, vaa.ChainIDTerra).Run); err != nil {
+			cosmwasm.NewWatcher(*terraWS, *terraLCD, *terraContract, lockC, setC, chainObsvReqC[vaa.ChainIDTerra], common.ReadinessTerraSyncing, vaa.ChainIDTerra).Run); err != nil {
```

Comment:
I'm not sure if this is a thing, but is cosmwasm version 1.0.0 and only applies to Terra2 and Injective?  Andy old Terra should stay on the old version of cosmwasm?  Does that apply here?